dashboard/test/controllers/maker_controller_test.rb

```diff
@@ -4,11 +4,48 @@ class MakerControllerTest < ActionController::TestCase
   include Devise::Test::ControllerHelpers
 
   setup do
+    @student = create :student
     @teacher = create :teacher
     @admin = create :admin
     @school = create :school
   end
 
+  test "home redirects to sign-in when user is signed out" do
+    assert_queries 0 do
```

Comment:
There isn't really a need to assert this if the redir happens

Reply:
Sounds good, I'll simplify this cases.  I was modeling it on this test in `HomeControllerTest`:

https://github.com/code-dot-org/code-dot-org/blob/dc6b354c5373c0ffa03821d8c1a4ca358f5aa069/dashboard/test/controllers/home_controller_test.rb#L73-L79

But this page will get a lot less traffic than `HomeController#index`, so maybe we don't care that much anyway.